documentation/src/main/asciidoc/reference/search-dsl-predicate.asciidoc

```diff
@@ -784,11 +784,78 @@ but can be <<search-dsl-predicate-common-boost,boosted>>,
 either on a per-field basis with a call to `.boost(...)` just after `.field(...)`/`.fields(...)`
 or for the whole predicate with a call to `.boost(...)` after `.matchingAny(...)` or `.matchingAll(...)`.
 
+[[search-dsl-predicate-and]]
+// TODO YB: find references to _combining_queries and add references to _anding_queries
+== [[_anding_queries]] `and`: combine predicates as a conjunction
+
+The `and` predicate matches documents that match a conjunction of inner predicates, called "clauses".
+`and` clauses are required to match: if they don't match, then the `and` predicate will not match.
```

Comment:
WDYT of that? Seems more straightforward to me.

```suggestion
The `and` predicate matches documents that match **all** of its inner predicates, called "clauses".
```

Reply:
Perfect!